fallregister_backend/src/main/java/de/coronavirus/application/controllers/api/AutocompleteApi.java

```diff
@@ -0,0 +1,50 @@
+package de.coronavirus.application.controllers.api;
+
+import de.coronavirus.domain.model.Address;
+import de.coronavirus.domain.model.Diagnosis;
+import de.coronavirus.domain.model.EmailAddress;
+import de.coronavirus.domain.model.PhoneNumber;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.List;
+
+@Api(tags = {"autocomplete"}, description = "Corona-Virus Autocomplete API")
+@RequestMapping("/corona")
```

Comment:
```suggestion
@RequestMapping("/autocomplete")
```
Du musst hier das RequestMapping der API anpassen.